config/creatrs/imp.cfg

```diff
@@ -87,6 +87,8 @@ LoyaltyLevel = 0
 DamageToBoulder = 30
 ; Creature thing size - XY and YZ; one cube is 256x256
 ThingSize = 256 512
+; The object the creature will create to use as a lair.
+LairObject = LAIR_IMP
```

Comment:
is there such thing as a LAIR_IMP? also maybe add a note that the object needs to be configured both ways

Reply:
Yes, it's simply one of the existing objects. The imp just never makes a lair. It looks like a dragons lair.